hello-world/src/main/resources/archetype-resources/src/main/resources/HelloWorldBuilder/help-useFrench_es.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    ¿Utilizar el franc\u00E9s?
```

Comment:
@rsandell should `¿` be converted to unicode?

Reply:
Or vice-versa: the help file is assumed to be UTF-8 I think.

Reply:
I had to put most special things in unicode, even in the html files.
Even if it might be unnecessary it won't hurt to do it.

Reply:
`\u00E9` is not valid HTML.